octavia-cli/tests/integration/test_generate/expected_rendered_yaml/destination_postgres/expected.yaml

```diff
@@ -0,0 +1,32 @@
+# Configuration for airbyte/destination-postgres
+# Documentation about this connector can be found at https://docs.airbyte.io/integrations/destinations/postgres
+resource_name: my_postgres_destination
+definition_type: destination
+definition_id: foobar
+definition_image: airbyte/destination-postgres
+definition_version: 0.3.13
+
+# EDIT THE CONFIGURATION BELOW!
+configuration:
+  host: # REQUIRED | string | Hostname of the database.
+  port: 5432 # REQUIRED | integer | Port of the database. | Example: 5432
+  database: # REQUIRED | string | Name of the database.
+  schema: "public" # REQUIRED | string | The default schema tables are written to if the source does not specify a namespace. The usual value for this field is "public". | Example: public
+  username: # REQUIRED | string | Username to use to access the database.
+  password: # 🤫 | OPTIONAL | string | Password associated with the username.
+  ssl: # OPTIONAL | boolean | Encrypt data using SSL.
+  tunnel_method:
+    ## Pick one valid structure among the examples below:
+    tunnel_method: "NO_TUNNEL" # REQUIRED | string | No ssh tunnel needed to connect to database
+    ## Another valid structure for tunnel_method:
```

Comment:
This feels a little bit hard to read to me, I think mainly because the "structural" comments like the one on this line are only distinguished with a second `#`. 

I think something like the following would be a bit more readable:
```
    ## -------- Pick one valid structure among the examples below: --------
    tunnel_method: "NO_TUNNEL" # REQUIRED | string | No ssh tunnel needed to connect to database
    ## -------- Another valid structure for tunnel_method: --------
    # tunnel_method: "SSH_KEY_AUTH" # REQUIRED | string | Connect through a jump server tunnel host using username and ssh key
    # tunnel_host: # REQUIRED | string | Hostname of the jump server host that allows inbound ssh tunnel.
    # tunnel_port: 22 # REQUIRED | integer | Port on the proxy/jump server that accepts inbound ssh connections. | Example: 22
    # tunnel_user: # REQUIRED | string | OS-level username for logging into the jump server host.
    # ssh_key: # 🤫 | REQUIRED | string | OS-level user account ssh key credentials in RSA PEM format ( created with ssh-keygen -t rsa -m PEM -f myuser_rsa )
    ## -------- Another valid structure for tunnel_method: --------
    # tunnel_method: "SSH_PASSWORD_AUTH" # REQUIRED | string | Connect through a jump server tunnel host using username and password authentication
    # tunnel_host: # REQUIRED | string | Hostname of the jump server host that allows inbound ssh tunnel.
    # tunnel_port: 22 # REQUIRED | integer | Port on the proxy/jump server that accepts inbound ssh connections. | Example: 22
    # tunnel_user: # REQUIRED | string | OS-level username for logging into the jump server host
    # tunnel_user_password: # 🤫 | REQUIRED | string | OS-level password for logging into the jump server host
```
This is just a suggestion so I will leave it up to you to decide what you think looks the best, but I think something like this is helpful for visually separating the different options that the user has

Reply:
Nice suggestion, I made the change.